dashi/poolqc/pooling_qc_sample.py

```diff
@@ -132,74 +210,71 @@ def update_title(lane_value, run_value):
     return 'You have selected lane {} in run {}'.format(lane_value, run_value)
 
 
+@app.callback(
+    dep.Output('Filter_drawer', 'open'),
+    [dep.Input('filters', 'n_clicks')]
+)
+def open_project_drawer(n_clicks):
+    return n_clicks is not None
+
+
+@app.callback(
+    dep.Output('index_threshold', 'value'),
+    [dep.Input('select_a_run', 'value'),
+     dep.Input('lane_select', 'value')]
+)
+def initial_threshold_value(run_alias, lane_alias):
+    run = df[(df['Run'] == run_alias) & (df['LaneNumber'] == lane_alias)]
+    run = run[~run['Run'].isna()].drop_duplicates('library')
```

Comment:
I don't understand the purpose of this line. `run = df[df['Run'] == run_alias]` above should already take care of this, as long as `run_alias` is not `NaN`.